lib/configuration/variables/sources/json-parse.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+
+const ServerlessError = require('../../../serverless-error');
+const { get } = require('lodash');
+
+module.exports = {
+  resolve: ({ address, isSourceFulfilled, params }) => {
+    if (!params && params[0] == null) {
+      throw new ServerlessError(
+        'Missing JSON object or JSON string in variable "jsonParse" source',
+        'MISSING_JSON_PARSE_JSON_OBJECT'
+      );
+    }
+
+    let jsonValue = {};
+    try {
+      jsonValue = typeof params[0] === 'string' ? JSON.parse(params[0]) : params[0];
+    } catch {
+      throw new ServerlessError(
+        'Invalid JSON object in variable "jsonParse" source',
+        'INVALID_JSON_PARSE_JSON_OBJECT'
+      );
+    }
+
+    return {
+      value: address ? get(jsonValue, address, '') : jsonValue,
```

Comment:
Let's not use `_.get`. Let's reproduce normal JS resolution, where attempting to resolve property value from not existing value will result with crash. Additionally if end value will be `undefined` we should fallback it to `null` (resolver cannot resolve `undefined` values)

Reply:
We want to support nested keys, but if nested structure is not found we should resolve with an error